charts/tractusx-connector-azure-vault/values.yaml

```diff
@@ -464,23 +507,27 @@ dataplane:
         issuer: ""
         # -- If preset enables certificate generation via cert-manager cluster-wide issuer
         clusterIssuer: ""
+
   # -- declare where to mount [volumes](https://kubernetes.io/docs/concepts/storage/volumes/) into the container
   volumeMounts:
   # -- [volume](https://kubernetes.io/docs/concepts/storage/volumes/) directories
   volumes:
+
   # -- [resource management](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) for the container
-  resources: {}
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
-  limits:
-    cpu: 1.5
-    memory: 1024Mi
-  requests:
-    cpu: 500m
-    memory: 128Mi
+  resources:
+    limits:
+      # -- Maximum CPU limit
+      cpu: 1.5
+      # -- Maximum memory limit
+      memory: 1024Mi
```

Comment:
What's the reason for dataplanes getting 1024Mib by default but control planes only get 512Mib?
Is this based only on "gut feeling" or on a use case/experience/measurement/empirical evidence of some sort?

Reply:
this needs to be answered by @saschaisele-zf. He configured that initially within https://github.com/eclipse-tractusx/tractusx-edc/pull/794

Reply:
The values came from experience with the EDC deployment we (ZF) had at that date.

Reply:
@paullatzelsperger can we close this conversation and merge the PR?

Reply:
I am generally ok with it, but we either use identical resources, or we put that information somewhere: documentation, a source comment,.... because the next developer will not have that "hidden" empirical knowledge

Reply:
Then let's just use the same value ;-)

Reply:
done. Please feel free to resolve the conversation and reapprove ;-)